retrofit/src/main/java/retrofit/RequestBuilder.java

```diff
@@ -381,6 +379,12 @@ Request build() {
     }
 
     RequestBody body = this.body;
+    if (formEncodingBuilder != null) {
```

Comment:
Their mutual exclusivity is why it's an `else`. If one matches I don't want to pay the cost of the field lookup for the other.